.circleci/workflows.yml

```diff
@@ -1740,31 +1740,6 @@ jobs:
       - run:
           name: Build
           command: yarn workspace @cypress/webpack-preprocessor build
-      - run:
-          name: Test babelrc
-          command: yarn test
-          working_directory: npm/webpack-preprocessor/examples/use-babelrc
-      - run:
-          name: Build ts-loader
-          command: yarn install
-          working_directory: npm/webpack-preprocessor/examples/use-ts-loader
-      - run:
-          name: Types ts-loader
-          command: yarn types
-          working_directory: npm/webpack-preprocessor/examples/use-ts-loader
-      - run:
-          name: Test ts-loader
-          command: yarn test
-          working_directory: npm/webpack-preprocessor/examples/use-ts-loader
-      - run:
-          name: Start React app
```

Comment:
Looks good, I looked at system tests, I see 

<img width="407" alt="image" src="https://user-images.githubusercontent.com/19196536/213063743-5368095e-e24d-418f-ac64-12861855df71.png">


We've also got a few system-tests using `.babelrc` so I don't think we are losing any coverage.